inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/manager/PushPrimerTrackerManager.kt

```diff
@@ -15,6 +15,10 @@ internal object PushPrimerTrackerManager {
         params[InAppMessagingConstants.RAT_EVENT_CAMP_ID] = campaignId
         params[InAppMessagingConstants.RAT_EVENT_SUBS_ID] = HostAppInfoRepository.instance().getSubscriptionKey()
         params[InAppMessagingConstants.RAT_EVENT_KEY_PERMISSION] = permission
+        params[InAppMessagingConstants.RAT_EVENT_KEY_CP] = mapOf(
```

Comment:
Do you need to add `cp` key here? I thought every key from `params` is put into `cp` by RAT SDK